src/MediaWiki.ts

```diff
@@ -249,6 +280,14 @@ class MediaWiki {
     return textDir;
   }
 
+  public getMainPage = function(url?: string): string {
```

Comment:
Why does it take an argument? It should not, the mainPage comes from `getSiteInfo()`.